assets/styles/modernism.css

```diff
@@ -252,3 +253,9 @@ a.link:after {
     font-size: 18px;
   }
 }
+
+::-webkit-scrollbar {
```

Comment:
oh shoot i totally goofed and forgot to remove this before comitting. I think I was trying to figure out an implementation where the inline scrolling was more distinguished so that users would easily tell that it was an inline scroll area. It was taking me too long to figure out so I abandoned the idea and settled with the current in-line scrolling (which after reading your feedback, [the inline-scrolling link you showed me](https://baymard.com/blog/inline-scroll-areas), and thinking about it, I agree, we should probably avoid inline scrolling overall and remove inline scrolling in general